src/cli/metric_collector.rs

```diff
@@ -0,0 +1,44 @@
+use crate::cli::Metrics;
```

Comment:
curious what the reasoning is to include this here in the `cli` mod vs the `metrics` mod?

Reply:
this collector is dumb and only works when running things from the CLI because it dumps everything to `stderr`